src/main/java/io/singularitynet/MissionHandlers/ObservationFromBigGridImplementation.java

```diff
@@ -0,0 +1,186 @@
+package io.singularitynet.MissionHandlers;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import io.singularitynet.MissionHandlerInterfaces.ICommandHandler;
+import io.singularitynet.MissionHandlerInterfaces.IObservationProducer;
+import io.singularitynet.utils.JSONWorldDataHelper;
+import io.singularitynet.projectmalmo.GridDefinition;
+import io.singularitynet.projectmalmo.MissionInit;
+import io.singularitynet.projectmalmo.ObservationFromBigGrid;
+import io.singularitynet.projectmalmo.ObservationFromBigGrd;
+import net.minecraft.block.BlockState;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.registry.Registries;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class ObservationFromBigGridImplementation extends HandlerBase implements IObservationProducer, ICommandHandler {
```

Comment:
Current name suggests that this observation will contain big grid, maybe better name will be ObservationFindBlock

Reply:
> ObservationFindBlock
Yeah. And it corresponds to `findBlock` in Mineflayer

Reply:
@noskill  Done